Automattic/jetpack
# [JS-Packages] Connection: Update RequiredPlan login button style

Fixes #22278 

#### Changes proposed in this Pull Request:

With the updates to the action button [here](https://github.com/Automattic/jetpack/commit/8e1d504bf44ef00e8f66106db6958c6c5feaf3ff)
the overrides in the Connection UI stopped working.  This patch updates them match the specificity of the action button update.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:






* Launch the Jetpack Backup plugin in a disconnected state and view the admin page to ensure the Login link is correctly aligned.

Before:
![image](https://user-images.githubusercontent.com/1992658/148809972-4a5df3a5-f83f-4691-a7f8-124be0c0aec7.png)

After:
<img width="1212" alt="Screen Shot 2022-01-10 at 12 22 01 PM" src="https://user-images.githubusercontent.com/1992658/148810004-e0cbd486-fb1a-4b7e-a496-22c588c81396.png">
